### PR TITLE
Determine how to mark as transferred in FOLIO system - close #8

### LINKED ISSUE #8: Determine how to mark as transferred in FOLIO system

### PR DESCRIPTION
**Pull request by GoZaddy**
## Pre-Merge Checklist:
Before merging this PR, please go through the following list and take appropriate actions.

- Does this PR meet or exceed the expected quality standards?
  - [ ] Code coverage on new code is 80% or greater
  - [ ] Duplications on new code is 3% or less
  - [ ] Check logging
  - [ ] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] Were any API paths or methods changed, added or removed?
  - [ ] Were there any schema changes?
  - [ ] Did any of the interface versions change?
  - [ ] Were permissions changed, added, or removed?
  - [ ] Are there new interface dependencies?
  - [ ] There are no breaking changes in this PR.
  
If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?  
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.  

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

File: src/main/java/org/folio/dew/batch/bursarfeesfines/service/impl/BursarExportServiceImpl.java [hunk 1]
```diff
@@ -80,13 +86,59 @@ private BursarExportJob extractBursarFeeFines(JobParameter jobParameter)
...
   public void transferAccounts(
-    List<Account> accounts,
+    List<AccountWithAncillaryData> accounts,
     BursarExportJob bursarFeeFines
   ) {
-    log.error("I don't know how to create transfer requests yet...");
-    // var transferRequest = toTransferRequest(accounts, bursarFeeFines);
-    // log.info("Creating {}.", transferRequest);
-    // bulkClient.transferAccount(transferRequest);
+    Set<AccountWithAncillaryData> transferredAccountsSet = new HashSet<>();
```
Comment (ncovercash) [inline src/main/java/org/folio/dew/batch/bursarfeesfines/service/impl/BursarExportServiceImpl.java:92]: Love the way you're handling this!

File: src/main/java/org/folio/dew/batch/bursarfeesfines/service/impl/BursarExportServiceImpl.java [hunk 2]
```diff
@@ -80,13 +86,59 @@ private BursarExportJob extractBursarFeeFines(JobParameter jobParameter)
...
+          )
+        )
+        .collect(Collectors.toList());
+      transferredAccountsSet.addAll(accountsToBeTransferred);
+
+      String accountName = getTransferAccountName(
+        bursarExportTransferCriteriaConditionsInner.getAccount().toString()
+      );
+
+      log.info(
```
Comment (ncovercash) [inline src/main/java/org/folio/dew/batch/bursarfeesfines/service/impl/BursarExportServiceImpl.java:115]: Please check that there are > 0 accounts for each condition before making transferRequests

**Review by ncovercash — CHANGES_REQUESTED** [review]
Looks great, just one small logic thing!

File: src/main/java/org/folio/dew/batch/bursarfeesfines/service/impl/BursarExportServiceImpl.java (hunk 1 above)
Comment (GoZaddy) [reply src/main/java/org/folio/dew/batch/bursarfeesfines/service/impl/BursarExportServiceImpl.java:92]: Thanks!

**Review by ncovercash — APPROVED** [review]
LGTM!  Thanks for your hard work on all this

**Pull request merged**